beacon_node/network/src/sync/range_sync/batch.rs

```diff
@@ -525,3 +530,29 @@ impl<E: EthSpec> std::fmt::Debug for BatchState<E> {
         }
     }
 }
+
+impl<E: EthSpec> BatchState<E> {
+    /// Creates a unicode visualisation for the batch state to display in logs for quicker and
+    /// easier recognition
+    ///
+    /// The current icons are:
+    /// - Empty/Uninitialized: E
+    /// - Downloading: D
+    /// - Awaiting Download: d
+    /// - Awaiting Validation: v
+    /// - Failed: F
+    /// - AwaitingProcessing: p
+    /// - Processing(_) => "P",
+    /// - Poisoned => "X",
```

Comment:
This mapping comment should be removed as it does not add any new information not visible in the code below and risks going stale